x/community/types/params.go

```diff
@@ -1,26 +1,47 @@
 package types
 
-import "time"
+import (
+	"errors"
+	fmt "fmt"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+)
 
 var (
 	DefaultUpgradeTimeDisableInflation = time.Time{}
+	// DefaultStakingRewardsPerSecond is ~4.6 KAVA per block, 6.3s block time
+	DefaultStakingRewardsPerSecond = sdkmath.NewInt(744191)
```

Comment:
might be worth mentioning this new default in the changelog as this will be the new inflation for the default chain going forward.